src/Task_3/Switch_with_places.java

```diff
@@ -0,0 +1,28 @@
+package Task_3;
+
+import java.util.Scanner;
+
+public class Switch_with_places {
+
+    public static void main(String args[]){
+
+        Scanner readeri = new Scanner(System.in);
+        System.out.println("Enter i: ");
+        int i = readeri.nextInt();
+
+        Scanner readerj = new Scanner(System.in);
```

Comment:
Scanner can be instantiated only once. 
You don't need to create it separately for 'i' and 'j', simply use the same scanner to get the `nextInt()'